spread.yaml

```diff
@@ -9,7 +9,6 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
```

Comment:
I see the benefit that this ensures that everything works correctly when we do re-exec (which will be the default for most people that it does re-exec). We can test with SNAP_REEXEC=0 here of course, I think this would actually test slightly less because this way we don't know that everything still works when it does the re-exec operation.